QLoggerWriter.h

```diff
@@ -22,7 +22,7 @@
  ** Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
  ***************************************************************************************/
 
-#include <QLoggerLevel.h>
+#include "QLoggerLevel.h"
```

Comment:
Revert to angle brackets.

Reply:
No

Reply:
Yes. What you have is a CMake configuration issue. C++ shouldn't not change because problems in external tools. See the comment [here](https://github.com/francescmm/QLogger/pull/12#discussion_r922822975).